editor.planx.uk/src/@planx/components/shared/Schema/InputFields/SelectInputField.tsx

```diff
@@ -10,17 +10,18 @@ import { FieldInputDescription } from "./shared";
 
 export const SelectFieldInput: React.FC<Props<QuestionField>> = (props) => {
   const { data, formik } = props;
-  const { id, errorMessage, name, value } = getFieldProps(props);
+  const { id, errorMessage, name, value, required, title } =
+    getFieldProps(props);
```

Comment:
We now need to grab `title` from the `getFieldProps()` helper function instead of `data.title` as this will be the formatted value (`" (optional)"` is appended).

I couldn't really find a simple way using TS (or anything else) to block a well-intentioned dev from incorrectly grabbing the raw `data`. We need it elsewhere and mistakes should be fairly obvious if we go wrong here.